addons/payment_adyen/models/payment_transaction.py

```diff
@@ -194,3 +195,55 @@ def _adyen_tokenize_from_feedback_data(self, data):
         _logger.info(
             "created token with id %s for partner with id %s", token.id, self.partner_id.id
         )
+
+    def _send_refund_request(self, **kwargs):
+        """ Override of payment to send a refund request to Adyen.
+        Note: self.ensure_one()
+        :return: None
+        """
+        super()._send_refund_request(**kwargs)
+        if self.provider != 'adyen':
+            return
+
+        refund_amount = kwargs.get('refund_amount', 0)
+
+        converted_amount = payment_utils.to_minor_currency_units(refund_amount,
+            self.currency_id, CURRENCY_DECIMALS.get(self.currency_id.name))
+
+        data = {
+            'merchantAccount': self.acquirer_id.adyen_merchant_account,
+            'modificationAmount': {
+                'value': converted_amount,
+                'currency': self.currency_id.name,
+            },
+            'originalReference': self.acquirer_reference,
+            'reference': self.reference,
+        }
+        response_content = self.acquirer_id._adyen_make_request(
+            url_field_name='adyen_payment_api_url',
+            endpoint='/refund',
+            payload=data,
+            method='POST'
+        )
+
+        response_content['merchantAccount'] = self.acquirer_id.adyen_merchant_account
```

Comment:
Is this line useful ? The request is already done and I don't see any use of the `merchantAccount` below :thinking: